packages/core-data/src/actions.js

```diff
@@ -96,3 +100,30 @@ export function receiveEmbedPreview( url, preview ) {
 		preview,
 	};
 }
+
+/**
+ * Action triggered to save an entity record.
+ *
+ * @param {string} kind    Kind of the received entity.
+ * @param {string} name    Name of the received entity.
+ * @param {Object} record  Record to be saved.
+ *
+ * @return {Object} Updated record.
+ */
+export function* saveEntityRecord( kind, name, record ) {
+	const entities = yield getKindEntities( kind );
+	const entity = find( entities, { kind, name } );
+	if ( ! entity ) {
+		return;
+	}
+	const key = entity[ key ] || 'id';
```

Comment:
There's now three instances of `|| 'id'` magic string in `core-data`. Makes me think it should be either encompassed in a `getEntityKey` utility function or defined as a constant `DEFAULT_KEY`.

Reply:
In authoring above comment, I realized we're referring to the value of `key`... during its own assignment. As used elsewhere, this should be `entityConfig.key`.

Aside: Would that be `entity.key` ? I'm often confused about entities vs. entity configurations.

Reply:
Above failing case should be covered by a test.

Reply:
There's no such thing as "entityConfig", it's used in the reducer to avoid an ESlint error (shadow variable).